motionplan/motionPlanner.go

```diff
@@ -72,7 +72,13 @@ func PlanRobotMotion(ctx context.Context,
 	constraintSpec *pb.Constraints,
 	planningOpts map[string]interface{},
 ) ([]map[string][]frame.Input, error) {
-	seedMap, _, err := framesystem.RobotFsCurrentInputs(ctx, r, fs)
```

Comment:
We can, but that turns a no-output change into a breaking change

Reply:
Not even, just changes the signature for a public function. I think you're right, it's likely worthwhile.

Reply:
Actually no, since other methods have their seed map passed in. This is the only one using `robot`, and it's just a convenience function that isn't used.